Changed section on database properties to reflect they must be abstract (#268)
Starcounter/Docs: docs/database-classes.md

## Fields and properties

Database classes should only use properties - either **abstract** or with an explicitly **declared body**. Abstract properties should be public and have a data type that is supported by the database.

> Properties that is neither abstract nor defines a body will generate an error.

Reviewer: ```suggestion
Any property that is neither abstract nor defines a body will generate an error.
```

```csharp

> class Supported

Reviewer: ```suggestion
abstract class Supported
```

> abstract class Supported

Reviewer: ```suggestion
public abstract class Supported
```

{

>   public abstract string Name { get; set; }

Reviewer: ```suggestion
    public abstract string Name { get; set; }
```

>   public string CapitalName { get => Name.ToUpper(); }

Reviewer: ```suggestion
    public string CapitalName { get => Name.ToUpper(); }
```

}
```

```csharp

> abstract class NotSupported

Reviewer: ```suggestion
public abstract class NotSupported
```

{

>   public abstract System.IO.File File { get; set; }  // Data type not supported

Reviewer: ```suggestion
    public abstract System.IO.File File { get; set; }  // Data type not supported
```

>   public string Name { get; set; }                   // Concrete - must be abstract

Reviewer: ```suggestion
    public string Name { get; set; }                   // Concrete - must be abstract
```

>   public virtual int Age { get; set; }               // Virtual - must be abstract

Reviewer: ```suggestion
    public virtual int Age { get; set; }               // Virtual - must be abstract
```
